com.unity.shadergraph/Editor/GraphUI/GraphElements/ModelUI/RedirectNode.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Linq;
+using System.Text;
+using UnityEditor.GraphToolsFoundation.Overdrive;
+using UnityEditor.ShaderGraph.GraphUI.DataModel;
+using UnityEditor.ShaderGraph.GraphUI.Utilities;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.ShaderGraph.GraphUI.GraphElements
+{
+    public class RedirectNode : Node
+    {
+        protected override void BuildPartList()
+        {
+            PartList.AppendPart(InOutPortContainerPart.Create(portContainerPartName, Model, this, ussClassName));
+            AddToClassList("sg-redirect-node");
+            this.AddStylesheet("RedirectNode.uss");
+        }
+
+        protected override void BuildContextualMenu(ContextualMenuPopulateEvent evt)
+        {
+            base.BuildContextualMenu(evt);
+
+            // TODO: Remove
```

Comment:
What does this mean?  Remove what? And why?  (just wary of code going in with todo notes that someone other than the author might not understand)

Reply:
Oops, I didn't finish writing my sentence. It was for a temporary context menu item below. But on second look, it's probably not useful to check into staging -- I'll go ahead and remove it.